__tests__/app.test.js

```diff
@@ -41,10 +41,28 @@ describe("GET/api", () => {
     });
 })
 
-
-
-
-
+describe("GET/api/articles", () => {
+    test("200: gets all articles, with the body property removed, a commment count added and sorted by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+           const {articles} = body
+           articles.forEach((article) => {
```

Comment:
You're using a forEach which should immediately be a flag that you need to check this isn't a false positive and a empty array has been returned back. Remember to check this array actually has the right amount of articles inside it

Reply:
Noted, is it sufficient to check that at least 1 element is contained for robustness? i.e. if the test data changes for some reason?